04_hex_and_bytes.md

```diff
@@ -44,22 +44,33 @@ Wouldn't we expect this to look more like `00000001` so that it is correctly int
 Well, this gets us into the topic of endianness.
 
 ### Little Endian vs Big Endian
-Bytes can be stored in two different orders.
-When bytes are stored as little-endian, the least significant byte is stored first (in the lowest memory address).
-Modern CPUs usually store numbers as little endian.
-When bytes are stored as big-endian, the most significant byte is stored first.
-So when we look at a transaction, the version is defined by the protocol as being 4 bytes in *little endian*.
-This means the least significant position appears first.
-What do we mean by *position*? Well let's look at decimal numbers for comparison.
-
-When looking at decimal numbers, we typically read it from left to right with the most sigificant digit first to the least significant digit last.
+Bytes can be stored in two different orders: little vs. big endian.
+When bytes are stored as little-endian, the least significant byte is stored in the lowest memory address.
+That's how modern CPUs usually store numbers which make sense from a logical engineering perspective.
+When bytes are stored as big-endian, it's the opposite: the most significant byte is stored in the lowest memory address.
+
+In its heart, Bitcoin is a communication protocol.
+For communication protocols, the different endianness are crucial to understand because bytes will be sent in a specific sequence.
```

Comment:
How about "For communication protocols, endianness is crucial to understand because bytes will be sent in a specific sequence."